### PR TITLE
add reverse sort and fix .millis issue

### PR DESCRIPTION
**Pull request by webb-e**

### CODE REVIEW

**Comment by webb-e** [comment]
@googlebot I signed it!

**Review by jdbcode — CHANGES_REQUESTED** [review]
Thanks for changes! There are just two minor notes on comments. You can edit [your file](https://github.com/webb-e/earthengine-community/edit/master/tutorials/identifying-first-day-no-snow/index.md?pr=%2Fgoogle%2Fearthengine-community%2Fpull%2F299) and commit the changes to the master branch to address the comments.

File: tutorials/identifying-first-day-no-snow/index.md [hunk 1]
```diff
@@ -254,7 +256,8 @@ var annualList = years.map(function(year) {
   var noSnowImg = yearCol
     // Add date bands to all images in this particular collection.
     .map(addDateBands)
-    // Sort the images by time.
+    // Sort the images by time. 
+    //Use .sort('millis' false) to reverse sort (find first day of snow in the fall)
```
[the remark this answers is not in the document]
Comment (webb-e) [reply tutorials/identifying-first-day-no-snow/index.md]: Ok let me know if this works for you.

**Comment by jdbcode** [comment]
Looks great - thanks! I'm going to make two small editorial changes for consistency (add period and single space).

**Review by jdbcode — APPROVED** [review]
Closes issue #298

**Pull request merged**